zipline/data/bundles/quandl.py

```diff
@@ -118,46 +118,53 @@ def gen_asset_metadata(data, show_progress):
     if show_progress:
         log.info('Generating asset metadata.')
 
-    asset_metadata = data.groupby(
+    data = data.groupby(
         by='symbol'
     ).agg(
         {'date': [np.min, np.max]}
-    ).reset_index()
-    asset_metadata['start_date'] = asset_metadata.date.amin
-    asset_metadata['end_date'] = asset_metadata.date.amax
-    del asset_metadata['date']
-    asset_metadata.columns = asset_metadata.columns.get_level_values(0)
-
-    asset_metadata['exchange'] = 'QUANDL'
-    asset_metadata['auto_close_date'] = \
-        asset_metadata['end_date'].values + pd.Timedelta(days=1)
-    return asset_metadata
+    )
+    data.reset_index(inplace=True)
+    data['start_date'] = data.date.amin
+    data['end_date'] = data.date.amax
+    del data['date']
+    data.columns = data.columns.get_level_values(0)
+
+    data['exchange'] = 'QUANDL'
+    data['auto_close_date'] = \
```

Comment:
Minor thing, but I don't think the line needs to be wrapped any more.

Reply:
oh you are right. I renamed the DataFrame here and it gave me just enough space to fit it in one line.